drivers/soundwire/slave.c

```diff
@@ -32,7 +32,7 @@ static int sdw_slave_add(struct sdw_bus *bus,
 
 	/* Initialize data structure */
 	memcpy(&slave->id, id, sizeof(*id));
-	slave->dev.parent = bus->dev;
+	slave->dev.parent = &bus->md->dev;
```

Comment:
again you don't need to change this if bus->dev is set to &bus->md->dev

Reply:
Now md->dev is a child of bus->dev. And when we call ``pm_runtime_get_sync(slave->bus->dev);`` in sdw_nread/nwrite  what we really want is to wake up the intel device not master device, right?